ftva_lab_data/management/commands/import_status_info.py

```diff
@@ -0,0 +1,139 @@
+        ),
+    ]
+
+    status_id_list = []
+    for substring, status_id in status_info_to_model_id_map:
+        if substring.lower() in status_info.lower():
+            status_id_list.append(status_id)
+    # Pandas wants a hashable type to make dropping duplicates possible,
+    # so return a tuple rather than a list
+    return tuple(status_id_list)
+
+
+def match_record(row: pd.Series):
+    """Try to get a matching `SheetImport` object using data from the Google Sheet."""
+
+    # These three fields alone uniquely match most (6460 out of 6741)
+    # of the unique rows with status info in the `Copy of DL` sheet.
+    # Using `filter()` over `get()` provides more flexibility to operate
+    # on the return value in the calling `handle` function.
+    file_folder = str(row["File Folder Name"]).strip()
+    sub_folder = str(row["Sub Folder"]).strip()
+    file_name = str(row["File Name"]).strip()
+
+    return SheetImport.objects.filter(
+        file_folder_name=file_folder,
+        sub_folder_name=sub_folder,
+        file_name=file_name,
+    )
+
+
+class Command(BaseCommand):
+    help = "Import status information from `Copy of DL Sheet_10_18_2024` Google Sheet"
+
+    def add_arguments(self, parser) -> None:
+        parser.add_argument(
+            "-f",
+            "--file_name",
+            type=str,
+            required=True,
+            help="Path to XLSX copy of Google Sheet",
+        )
+
+    def handle(self, *args, **options) -> None:
+        file_name = options["file_name"]
+        tapes_data = pd.read_excel(
+            file_name,
+            sheet_name="Tapes(row 4560-24712)",  # NOTE: sheet name is hard-coded here
+        )
+
+        # Provide total row count for tapes data
+        total_rows = len(tapes_data.index)
+        print(f"There are {total_rows} rows in source data.")
+
+        # # Count duplicated rows, then drop them
+        duplicate_count = tapes_data.duplicated().sum()
+        print(
+            f"Found {duplicate_count} duplicated rows in source data. Dropping them..."
+        )
+        tapes_data.drop_duplicates(inplace=True)
+
+        # Count rows without status info, then drop them
+        no_status_info_count = tapes_data["Requires Manual Intervention"].isna().sum()
+        print(
+            f"Found {no_status_info_count} rows without status info in source data.",
+            "Dropping them...",
+        )
+        tapes_data.dropna(subset=["Requires Manual Intervention"], inplace=True)
+
+        # Create a new column of status IDs parsed from status information
+        tapes_data["status_ids"] = tapes_data["Requires Manual Intervention"].apply(
+            parse_status_info
+        )
+
+        # Fill NA values with empty string, to avoid type issues with Django.
+        # Being explicit with columns, because Pandas complains otherwise.
+        tapes_data.fillna(
+            {"File Folder Name": "", "Sub Folder": "", "File Name": ""}, inplace=True
+        )
+
```

Comment:
Add a status message here, with a count of how many records are left - "Checking {remaining_records_count}..." or similar.  That way we don't have to add up the "updated" and problem counts :)